ddtrace/profiling/collector/_task.pyx

```diff
@@ -0,0 +1,34 @@
+from ddtrace import compat
+from ddtrace.internal import nogevent
+
+from . import _threading
+
+
+try:
+    import gevent._tracer
+except ImportError:
+    _gevent_tracer = None
+else:
+    # NOTE: bold assumption: this module is always imported by the MainThread.
+    # A GreenletTracer is local to the thread instantiating it and we assume this is run by the MainThread.
+    _gevent_tracer = gevent._tracer.GreenletTracer()
+
+
+cpdef get_task(thread_id):
```

Comment:
Is there much of a benefit to justify Cythonizing here (besides the base improvement we'd expect)? AFAICT it doesn't look like we'd gain anything too substantial 🤔